### PR TITLE
Add support for the .url shortcuts

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: docs/shortcuts.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+}
+```
+
+## GET /shortcuts/:id
+
+This route can be used to get information about a shortcut. A permission to
+read the file is required to use it. There are two variants, depending on the
+`Accept` header.
+
+### Request (JSON variant)
```
Comment (edas) [inline docs/shortcuts.md]: Does this simply duplicate the /files route ?

Comment (nono) [reply docs/shortcuts.md]: No, it also parses the content of the .url file to extract the URL

File: docs/shortcuts.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+          },
+          "app": "photos",
+          "_type": "io.cozy.files",
+          "mime": "image/jpg"
+        }
+      }
+    }
+  }
+}
+```
```
Comment (Crash--) [inline docs/shortcuts.md]: Can you add an example with an external link? Do we receive the external "link" in the `data.attributes.url` attribute ?

Comment (nono) [reply docs/shortcuts.md]: Yes, the `url` is filled by parsing the content of the file, and it will be here for an external link.

**Pull request merged**